providers/base/units/tracers/resource.pxu

```diff
@@ -0,0 +1,12 @@
+id: kernel_config_file
+plugin: resource
+_summary:
+ Kernel config file resource
+command:
+ if [[ -f "/boot/config-$(uname -r)" ]]
+ then
+     echo "state: supported"
+ else
+     echo "state: unsupported"
+ fi
+_description: Creates resource info for kernel config file
```

Comment:
More truthful description would be to state that it checks for kernel config file existence.